sdk/python/kfp/dsl/_component_bridge.py

```diff
@@ -11,91 +11,187 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Function for creating ContainerOp instance."""
 
 import copy
 from typing import Any, Mapping
-from ..components.structures import ComponentSpec, ComponentReference
-from ..components._components import _default_component_name, _resolve_command_line_and_paths
-from ..components._naming import _sanitize_python_function_name, generate_unique_name_conversion_table
-from .. import dsl
+
+from kfp import dsl
+from kfp.components._components import _default_component_name, _resolve_command_line_and_paths
+from kfp.components._naming import _sanitize_python_function_name, generate_unique_name_conversion_table
+from kfp.components.structures import ComponentSpec, ComponentReference
+from kfp.dsl import ir_types
+from kfp.dsl._container_op import ContainerOp
```

Comment:
optional nit: Try importing module instead of class, unless there'll be many 'lines too long' below.

Related is [1] and the following quote from [Python zen](https://www.python.org/dev/peps/pep-0020/)

> Namespaces are one honking great idea -- let's do more of those!

[1] https://google.github.io/styleguide/pyguide.html#22-imports